Source/WebCore/svg/graphics/filters/SVGFilter.cpp

```diff
@@ -86,7 +86,11 @@ static std::optional<std::tuple<SVGFilterEffectGraph, FilterEffectGeometryMap>>
     if (filterElement.countChildNodes() > maxCountChildNodes)
         return std::nullopt;
 
-    const auto colorSpace = filterElement.colorInterpolation() == ColorInterpolation::LinearRGB ? DestinationColorSpace::LinearSRGB() : DestinationColorSpace::SRGB();
+    auto firstPrimitive = childrenOfType<SVGFilterPrimitiveStandardAttributes>(filterElement).first();
+    const auto colorSpace = (firstPrimitive && firstPrimitive->colorInterpolation() == ColorInterpolation::SRGB)
+    ? DestinationColorSpace::SRGB()
+    : DestinationColorSpace::LinearSRGB();
```

Comment:
Nit: I *think* we would want lines 91 & 92 indented since they are continuations of the prior lines.